internal/bsr/internal/journal/journal.go

```diff
@@ -61,6 +62,11 @@ func (j *Journal) Record(op, f string) error {
 	return err
 }
 
+// WriteAndClose writes and closes the journal
+func (j *Journal) WriteAndClose(b []byte) (int, error) {
+	panic("not implemented")
+}
+
```

Comment:
Is this necessary?

Reply:
I don’t think it’s necessary. Took it out